src/core/client/stream/tabs/Comments/Comment/CommentContainer.tsx

```diff
@@ -428,6 +428,21 @@ export const CommentContainer: FunctionComponent<Props> = ({
                             </Flex>
                           </Button>
                         )}
+
+                        {comment.author &&
```

Comment:
Could be simplified by instead writing:

```suggestion
                        {comment.author?.avatar &&
```